src/components/Post/PostHeading.tsx

```diff
@@ -146,6 +147,7 @@ const PostHeading: FC<IPostHeadingProps> = (props) => {
 		})();
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, [api, apiReady]);
+	setRequestedAmount(formatBnBalance(String(requestedAmt), { numberAfterComma: 2, withUnit: true }, network));
```

Comment:
why we are doing this here

Reply:
yaha se requested amount pick karke govsidebar mai bounty chart mai use kar rahe hai